mx.py

```diff
@@ -2222,6 +2222,9 @@ def resolveDeps(self):
         if licenseId:
             self.theLicense = get_license(licenseId, context=self)
 
+    def substVars(self, text):
+        return text.format(**vars(self))
```

Comment:
Ok. At some later point when we document the suite.py format, we'll convey this information as well.